contracts/ColonyFunding.sol

```diff
@@ -276,8 +276,9 @@ contract ColonyFunding is ColonyStorage, PatriciaTreeProofs { // ignore-swc-123
   {
     ITokenLocking tokenLocking = ITokenLocking(IColonyNetwork(colonyNetworkAddress).getTokenLocking());
     uint256 totalLockCount = tokenLocking.lockToken(token);
-
-    require(activeRewardPayouts[_token] == 0, "colony-reward-payout-token-active");
+    uint256 thisPayoutAmount = sub(fundingPots[0].balance[_token], possiblyPendingRewardPayments[_token]);
```

Comment:
They're not all pending, was my reasoning, at the very least because payouts that people have waived their right to (and so are not pending being claimed) are included.... But maybe this was too conservative on my part. Will change.